Wrote about FMs (#40)
thesis20/thesis_paper: sections/experiments/experiments-chapter.tex

\subsection{Compared methods}
In the experiments, the CSGCN models are compared against the following methods for context-specific recommendations:
\begin{itemize}
    \item Factorization Machine (FM) \cite{fmrendle}
    \item Convolutional Factorization Machine (CFM) \cite{CFM}
    \item Neural Factorization Machine (NFM) \cite{NeuralFM}
    % \item Top Pop
    % \item CAMF \cite{CAMF}
\end{itemize}

> FMs are a relatively simple model that models all interactions between variables using factorized parameters.

Reviewer: ```suggestion
FM is a relatively simple model that models all interactions between variables using factorized parameters.
```
